src/js/edu-benefits/5490/helpers.jsx

```diff
@@ -1,8 +1,23 @@
+import _ from 'lodash/fp';
+
 import React from 'react';
 import { transformForSubmit } from '../../common/schemaform/helpers';
 
 export function transform(formConfig, form) {
-  const formData = transformForSubmit(formConfig, form);
+  // Clone the form in so we don't modify the original...because of reasons FP
+  const newForm = _.cloneDeep(form);
+
+  // Copy the data if necessary
+  if (form.sponsorInformation.data['view:currentSameAsPrevious']) {
+    newForm.sponsorInformation.data['view:currentSponsorInformation'] = {
+      veteranFullName: form.benefitHistory.data.previousBenefits.veteranFullName,
+      'view:veteranId': {
+        veteranSocialSecurityNumber: form.benefitHistory.data.previousBenefits.veteranSocialSecurityNumber
```

Comment:
Verified this is a good idea--separate ticket filed and added to current sprint: https://github.com/department-of-veterans-affairs/vets.gov-team/issues/1929